whylog/assistant/regex_assistant_exceptions.py

```diff
@@ -0,0 +1,7 @@
+class NoDateGroupException(Exception):
+    def __init__(self, line):
+        self.line = line
+        self.message = 'No date group in regex'
```

Comment:
constants should be put in class static variable, but in this case it is used only once, so it should be inlined in `__str__`. Also, the exception does not contain the line in question, which makes it much harder for the user to figure out which line is the problem. Please add the line in question to the `__str__` output.